components/scream/cime_config/namelist_defaults_scream.xml

```diff
@@ -0,0 +1,130 @@
+<?xml version="1.0"?>
+
+<namelist_defaults>
+
+  <file name="scream_input.yaml" format="yaml">
+    <Atmosphere Driver>
+      <Atmosphere Processes>
+        <Number of Entries>6</Number of Entries>
+        <Schedule Type>Sequential</Schedule Type>
+        <Process 0>
+          <Process Name>Homme</Process Name>
+          <Enable Output Fields Checks>False</Enable Output Fields Checks>
+          <Enable Input Fields Checks>False</Enable Input Fields Checks>
+          <Grid>Dynamics</Grid>
+        <\Process 0>
+        <Process 1>
+          <Process Name>SHOC</Process Name>
+          <Enable Output Fields Checks>False</Enable Output Fields Checks>
+          <Enable Input Fields Checks>False</Enable Input Fields Checks>
+          <Grid>Physics GLL</Grid>
+        <\Process 1>
+        <Process 2>
+          <Process Name>CldFraction</Process Name>
+          <Enable Output Fields Checks>False</Enable Output Fields Checks>
+          <Enable Input Fields Checks>False</Enable Input Fields Checks>
+          <Grid>Physics GLL</Grid>
+        <\Process 2>
+        <Process 3>
+          <Process Name>SPA</Process Name>
+          <Enable Output Fields Checks>False</Enable Output Fields Checks>
+          <Enable Input Fields Checks>False</Enable Input Fields Checks>
+          <Grid>Physics GLL</Grid>
+          <SPA Remap File>none</SPA Remap File>
+          <SPA Data File>${DIN_LOC_ROOT}/atm/scream/init/spa_file_unified_and_complete_ne4_scream.nc</SPA Data File>
+        <\Process 3>
+        <Process 4>
+          <Process Name>P3</Process Name>
+          <Enable Output Fields Checks>False</Enable Output Fields Checks>
+          <Enable Input Fields Checks>False</Enable Input Fields Checks>
+          <Grid>Physics GLL</Grid>
+        <\Process 4>
+        <Process 5>
+          <Process Name>RRTMGP</Process Name>
+          <Enable Output Fields Checks>False</Enable Output Fields Checks>
+          <Enable Input Fields Checks>False</Enable Input Fields Checks>
+          <Grid>Physics GLL</Grid>
+          <active_gases>h2o, co2, o3, n2o, co, ch4, o2, n2</active_gases>
+          <Orbital Year>-9999</Orbital Year>
+          <Orbital Eccentricity>-9999</Orbital Eccentricity>
+          <Orbital Eccentricity COMPSET=".*SCREAM%AQUA.*">0</Orbital Eccentricity>
+          <Orbital Obliquity>-9999</Orbital Obliquity>
+          <Orbital Obliquity COMPSET=".*SCREAM%AQUA.*">0</Orbital Obliquity>
+          <Orbital MVELP>-9999</Orbital MVELP>
+          <Orbital MVELP COMPSET=".*SCREAM%AQUA.*">0</Orbital MVELP>
+        <\Process 5>
+      <\Atmosphere Processes>
+      <Time Stepping>
+        <Time Step>300</Time Step>
+        <Start Time>12, 30, 0</Start Time>
+        <Start Date>2000, 2, 1</Start Date>
+        <Number of Steps>288</Number of Steps>
+      <\Time Stepping>
+      <Grids Manager>
+        <Type>Dynamics Driven</Type>
+        <Reference Grid>Physics GLL</Reference Grid>
+        <Dynamics Driven>
```

Comment:
Along the topic of simplifying our YAML files structure, this is another sublist that need not to exist. Unlike atm procs, we only build _one_ grids manager per run, so we don't need to have a sublist for each grids manager type, and we can just put all the params in the "Grids Manager" list.